packages/lib/src/interactions/box.test.ts

```diff
@@ -0,0 +1,203 @@
+import { Vector3 } from 'three';
+
+import Box from './box';
+
+const ZERO_COORDS = { x: 0, y: 0, z: 0 };
+
+describe('size', () => {
+  it('should compute box size', () => {
+    const box1 = Box.empty();
+    expect(box1.size).toEqual({ width: 0, height: 0 });
+
+    const box2 = Box.empty(new Vector3(10, 10));
+    expect(box2.size).toEqual({ width: 0, height: 0 });
+
+    const box3 = Box.empty().expandByPoint(new Vector3(10, -5));
+    expect(box3.size).toEqual({ width: 10, height: 5 });
+  });
+});
+
+describe('center', () => {
+  it('should compute box center', () => {
+    const box1 = Box.empty();
+    expect(box1.center).toMatchObject(ZERO_COORDS);
+
+    const box2 = Box.empty(new Vector3(10, 10));
+    expect(box2.center).toMatchObject({ x: 10, y: 10, z: 0 });
+
+    const box3 = Box.empty().expandByPoint(new Vector3(10, -5));
+    expect(box3.center).toMatchObject({ x: 5, y: -2.5, z: 0 });
+  });
+});
+
+describe('empty', () => {
+  it('should create empty box at (0, 0, 0)', () => {
+    const box = Box.empty();
+
+    expect(box.size).toEqual({ width: 0, height: 0 });
+    expect(box.center).toMatchObject(ZERO_COORDS);
+
+    // `new Box()` initializes min and max to (+∞, +∞, +∞)` and (-∞, -∞, -∞) respectively
+    expect(box.min).toMatchObject(ZERO_COORDS);
+    expect(box.max).toMatchObject(ZERO_COORDS);
+    expect(box.min).not.toBe(box.max); // different `Vector3` instances
+  });
+
+  it('should create empty box at given coordinates', () => {
+    const box = Box.empty(new Vector3(10, 20));
+
+    const expectedCoords = { x: 10, y: 20, z: 0 };
+    expect(box.size).toEqual({ width: 0, height: 0 });
+    expect(box.center).toMatchObject(expectedCoords);
+    expect(box.min).toMatchObject(expectedCoords);
+    expect(box.max).toMatchObject(expectedCoords);
+  });
+});
+
+describe('fromPoints', () => {
+  it('should create box that includes all given points', () => {
+    const box = Box.fromPoints(new Vector3(10, 10), new Vector3(-5, 12));
+    expect(box.min).toMatchObject({ x: -5, y: 10 });
+    expect(box.max).toMatchObject({ x: 10, y: 12 });
+  });
+
+  it('should create empty box when called with single point', () => {
+    const vec = new Vector3(-5, 12);
```

Comment:
I think it is best continuing your pattern of declaring the `box` rather than the `vec`
```suggestion
    const box = Box.fromPoints(new Vector3(-5, 12));
```